name.py

```diff
@@ -0,0 +1,3 @@
+name = input ("Escribe tu nombre carnalito: ")
+last_name = inputt ("Escribe tu apellido carnalito: ")
```

Comment:
te mamaste asi no es, no es inputt es input :D
```suggestion
last_name = input ("Escribe tu apellido carnalito: ")
```

Reply:
me vale cabrón, yo hago lo que me da la gana sin miedo a nada. Te voy a blokiar careverga. doxeado por pelón.

Reply:
pues bueno a volarle ingesurronia, como el cohete que te puse vuelale a la ver...